.jenkins/pytorch/common.sh

```diff
@@ -81,7 +81,7 @@ if which sccache > /dev/null; then
   # Save sccache logs to file
   sccache --stop-server || true
   rm ~/sccache_error.log || true
-  SCCACHE_ERROR_LOG=~/sccache_error.log RUST_LOG=sccache::server=error sccache --start-server
+  SCCACHE_ERROR_LOG=~/sccache_error.log SCCACHE_IDLE_TIMEOUT=1200 RUST_LOG=sccache::server=error sccache --start-server
```

Comment:
does this need to be in this PR?

Reply:
I'm putting it in a separate PR #16728, it's just here for now because I needed to verify that it works.